ccxt.js

```diff
@@ -13132,6 +13157,13 @@ var poloniex = {
         let timestamp = this.parse8601 (trade['date']);
         let id = undefined;
         let order = undefined;
+        let symbol = undefined;
+        if (!market) {
+            if ('currencyPair' in trade) {
+                let marketId = trade['marketId'];
```

Comment:
this should be:
```JavaScript
let marketId = trade['currencyPair']; // 'currencyPair' instead of 'marketId' here
```
as it seems it me. But, again, don't worry, I'll fix it.